rubocop.yml

```
@@ -110,6 +110,9 @@ Layout/HashAlignment:
 Layout/HeredocIndentation:
   Enabled: false
 
+Layout/LeadingCommentSpace:
+  AllowRBSInlineAnnotation: true
```

To enable this config without bumping the minimum RuboCop version, we can use the same technique as we did to configure new cops:

```suggestion
<% if rubocop_version >= "1.67" %>
Layout/LeadingCommentSpace:
  AllowRBSInlineAnnotation: true
<% end %>
```

---

On the other hand, _The Great Reset_ certainly warrants a major version bump, so we could take the opportunity to bump the minimum RuboCop version.

Thoughts @rafaelfranca?

---

We can do that later.